neuralop/losses/data_losses.py

```diff
@@ -132,32 +132,37 @@ def abs(self, x, y, h=None):
             
         return diff
 
-    def rel(self, x, y):
+    def rel(self, x, y, squared=True):
         """
-        rel: relative LpLoss
-        computes ||x-y||/||y||
+        rel: squared relative LpLoss
+        computes ||x-y||^2/||y||^2
 
         Parameters
         ----------
         x : torch.Tensor
             inputs
         y : torch.Tensor
             targets
+        squared : bool
+            whether to compute squared relative loss
+            by default True
         """
 
         diff = torch.norm(torch.flatten(x, start_dim=-self.d) - torch.flatten(y, start_dim=-self.d), \
                           p=self.p, dim=-1, keepdim=False)
         ynorm = torch.norm(torch.flatten(y, start_dim=-self.d), p=self.p, dim=-1, keepdim=False)
 
         diff = diff/ynorm
+        if squared: 
+            diff = diff ** 2
```

Comment:
We just just not apply the square root to begin with rather than re-square afterwards.